lib/consensus/isaac.go

```diff
@@ -151,16 +150,25 @@ func (is *ISAAC) isInitRound(round round.Round) bool {
 	return is.LatestRound.BlockHash == "" && round.BlockHeight == genesisHeight
 }
 
-func (is *ISAAC) startSync() {
-	height, nodeAddrs := is.getSyncInfo()
-	if is.syncer != nil && len(nodeAddrs) > 0 {
+func (is *ISAAC) StartSync(height uint64, nodeAddrs []string) {
+	is.log.Debug("begin is.StartSync")
+	if is.syncer != nil && len(nodeAddrs) > 0 && is.latestReqSyncHeight < height {
+		is.Node.SetSync()
+		is.log.Info("node state transits to sync")
+		is.latestReqSyncHeight = height
+		is.log.Debug("before is.SetSyncTargetBlock")
 		if err := is.syncer.SetSyncTargetBlock(context.Background(), height, nodeAddrs); err != nil {
 			is.log.Error("syncer.SetSyncTargetBlock", "err", err, "height", height)
 		}
+		is.Node.SetSync()
```

Comment:
Is there some meaning to set `is.Node.SetSync()` twice?

Reply:
@spikeekips thanks, it's leftover :)